themes/finna2/templates/record/labels.phtml

```diff
@@ -1,7 +1,23 @@
 <?php if (!empty($labels)): ?>
   <ul class="record-labels">
     <?php foreach ($labels as $label): ?>
-      <li class="label iconlabel <?=str_replace('_', '-', $label)?>"><?=$this->transEsc($label)?></li>
+      <?php
+        $tooltip = "RecordLabelTooltip:{$label}";
+        if ($this->R2()->isRegistered()) {
```

Comment:
Ei hyvä, että R2-juttua on sekaisin geneerissä templaatissa näin paljon. Saisiko sen eristettyä paremmin?

Reply:
Muutin käyttämään driver-kohtaista sivupohjaa, jonka renderoima sisältö asetetaan tooltippiin.